### DISCUSSION TITLE
Vector line delay

### DISCUSSION THREAD

**Discussion by jamesBrosnahan** [opentoonz/opentoonz#6461] [Ideas]
### Is your feature request related to a problem? Please describe

When drawing with smoothing on a vector layer there is a delay before seeing the line. See https://groups.google.com/d/msgid/opentoonz_en/b25811d0-fd56-4b67-a127-522d6871d6fen%40googlegroups.com?utm_medium=email&utm_source=footer

### Describe the solution you'd like

Can a preview animation be added to show a line being smoothed on the vector layer?
And can it be enabled like onion skin?

**Comment by RodneyBaker** [comment]
Any setting of smooth on vector levels adds delay due to the smoothing algorithm.
Any setting above zero triggers that additional calculation.

If your particular system is being too slow however it may be some other thing going.

Not entirely sure what you are requesting with the 'preview animation to add a line being smoothed'.
There is the Iron Tool that can be used to smooth vector lines.
Additional tools such as the Control Point Editor Tool allow us to interact with the control points and bias handles of vector lines.

**Comment by jamesBrosnahan** [comment]
Hi Rodney.

What I am saying is to draw a transparent preview of the original line with no smoothing as the cursor moves. Then have the smooth line appear as it is processed. The animation is the simulate an effect like paint setting into a canvas as you draw a stroke. This way people with slow systems do not have a lag between seeing what is drawn.

**Comment by RodneyBaker** [comment]
@jamesBrosnahan 
Please give the latest nightly release a try as that should speed up vector drawing on old and new systems alike.  

If the particular issue you are observing still exists in latest nightly perhaps that will help to hone in on the specific requirement more.
